### PR TITLE
Support week 8 rqa

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

**Comment by as2388** [comment]
@IsaackMwenda can you ask me to review this after merging #50?

**Comment by IsaackMwenda** [comment]
> @IsaackMwenda can you ask me to review this after merging #50?

@as2388 merged https://github.com/AfricasVoices/Project-ICRAF/pull/51 PTAL!

**Comment by as2388** [comment]
@IsaackMwenda can you merge master into this branch?

**Comment by IsaackMwenda** [comment]
> @IsaackMwenda can you merge master into this branch?

@as2388 merged PTAL.

File: run_scripts/4_coda_add.sh [hunk 1]
```diff
@@ -23,6 +23,8 @@ DATASETS=(
     "s01e05"
     "s01e06"
     "s01e07"
+    "s01e08"
+
```
Comment (as2388) [inline run_scripts/4_coda_add.sh:27]:
```suggestion
```

File: src/analysis_file.py [hunk 2]
```diff
@@ -147,7 +147,9 @@ def generate(user, data, csv_by_message_output_path, csv_by_individual_output_pa
             "radio_participation_s01e04",
             "radio_participation_s01e05",
             "radio_participation_s01e06",
-            "radio_participation_s01e07"
+            "radio_participation_s01e07",
+            "radio_participation_s01e08"
+
```
Comment (as2388) [inline src/analysis_file.py]:
```suggestion
```

File: src/lib/channels.py [hunk 3]
```diff
@@ -17,6 +17,8 @@ class Channels(object):
     S01E05_KEY = "radio_participation_s01e05"
     S01E06_KEY = "radio_participation_s01e06"
     S01E07_KEY = "radio_participation_s01e07"
+    S01E08_KEY = "radio_participation_s01e08"
+
```
Comment (as2388) [inline src/lib/channels.py:21]:
```suggestion
```
etc.

**Review by as2388 — APPROVED** [review]

**Pull request merged**